src/mon/FSCommands.cc

```diff
@@ -260,23 +259,18 @@ class SetHandler : public FileSystemCommandHandler
       }
 
       if (n <= 0) {
-        ss << "You must specify at least one MDS";
+        ss << "max_mds cannot be negative or zero";
```

Comment:
imo, the original message seems more intuitive, given `max_mds` doesn't seem to be a flag/value we're passing to the cli command but an internal meaning of what this value is.

Reply:
max_mds is a value we're passing on the cli.

ceph fs set <fs name> max_mds